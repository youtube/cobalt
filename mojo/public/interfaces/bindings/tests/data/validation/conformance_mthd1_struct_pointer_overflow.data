[dist4]message_header  // num_bytes
[u4]0                  // version
[u4]0                  // interface ID
[u4]1                  // name
[u4]0                  // flags
[u4]0                  // padding
[anchr]message_header

[dist4]method1_params   // num_bytes
[u4]0                   // version
[u8]0xFFFFFFFFFFFFFFFF  // param0: Test whether decoding the pointer causes
                        // overflow.
[anchr]method1_params
