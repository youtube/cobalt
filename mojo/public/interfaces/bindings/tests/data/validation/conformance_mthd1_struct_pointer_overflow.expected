VALIDATION_ERROR_ILLEGAL_POINTER
